qmt/data/geometry.py

```diff
@@ -151,6 +151,79 @@ def edge_coord_list(self, edge_name):
         coord_list = list(np.array(self.edges[edge_name].coords.xy).T)[:]
         return coord_list
 
+    def plot(
+        self,
+        axis_limits: Sequence[float] = [],
+        parts_to_exclude: Sequence[str] = [],
+        line_width: float = 20.0,
+        figsize: Sequence[float] = (10, 10),
+        colors: Sequence = list(mcd.XKCD_COLORS.values()),
+    ):
+        """
+        Plots the 2d geometry
+        :param axis_limits: Sequence with 4 values: xmin, ymin, xmax, ymax 
+        :param parts_to_exclude: Part/edge names that won't be plotted
+        :param line_width: Thickness of lines (only for edge lines)
+        :param figsize: Size of the figure
+        """
+        from matplotlib import pyplot as plt
+        import descartes
+
+        fig = plt.figure(figsize=figsize)
```

Comment:
Why not just give a figure (or better - axis of the figure) as an input? That way you can put the output as a part of some other figure.

Reply:
I wanted it to be as simple as possible so you can just call `.plot()` if you wanted to. I'll add that as an optional argument? If you don't pass it in, it'll create a new figure

Reply:
I guess axis = plt.figure().gca() as a default would work. Then you can drop fig_size etc

Reply:
But that wouldn't let you add the plot to another figure?

Reply:
It would. I mean if you don't give axis - this will make you a figure and an axis and if you do - then you can put it anywhere